src/test/javascript/spec/component/overview/course-lectures/course-lecture-details.component.spec.ts

```diff
@@ -188,6 +191,65 @@ describe('CourseLectureDetails', () => {
         downloadButton.nativeElement.click();
         expect(downloadAttachmentStub).toHaveBeenCalledOnce();
     }));
+
+    it('should set lecture unit as completed', fakeAsync(() => {
+        const lectureUnitService = TestBed.inject(LectureUnitService);
+        const completeSpy = jest.spyOn(lectureUnitService, 'setCompletion');
+        completeSpy.mockReturnValue(of(new HttpResponse<any>()));
+
+        courseLecturesDetailsComponent.lecture = lecture;
+        courseLecturesDetailsComponent.ngOnInit();
+        fixture.detectChanges();
+
+        expect(lectureUnit3.completed).toBeFalsy();
```

Comment:
```suggestion
        expect(lectureUnit3.completed).toBeFalse();
```
Is  there any chance that this value is undefined? If not i suggest to use `toBeFalse`

Reply:
Yes, this actually needs to be falsy. :)